nodebuilder/state/cmd/state.go

```diff
@@ -402,6 +416,75 @@ var queryRedelegationCmd = &cobra.Command{
 	},
 }
 
+var grantFeeCmd = &cobra.Command{
+	Use: "grant-fee [granteeAddress] [fee] [gasLimit]",
+	Short: "Grant an allowance to a specified grantee account to pay the fees for their transactions.\n" +
```

Comment:
We, by default, set allowance for all transactions, and it's not clear precisely what the term means, so I would remove allowance from the description.

Reply:
Grant allowance == give a permission. So, I think we can leave it as it is for now